app/bundles/CoreBundle/Model/FormModel.php

```diff
@@ -447,4 +448,11 @@ protected function flushAndCatch()
             );
         }
     }
+
+    public function getExportFilename(string $objectName): string
+    {
+        $date = (new DateTimeHelper())->toLocalString();
+
+        return str_replace(' ', '_', $date).'_'.InputHelper::alphanum($objectName, false, '_');
```

Comment:
This function doesn't fit here. The entity model should not be responsible for generating the export filename. It might be more appropriate to place this functionality in a helper class, such as `ExportHelper`.